src/userbase-js/src/api/auth.js

```diff
@@ -129,7 +129,7 @@ const getServerPublicKey = async () => {
 
     const method = 'GET'
     const url = `${config.getEndpoint()}/api/auth/server-public-key`
-    const timeout = timeout
+    const timeout = TEN_SECONDS_MS
```

Comment:
Just want to point this out. Haven't deeply investigated yet, but not sure why this didn't cause tests to fail earlier. Seems like there may be a caching issue going on both locally and with our CI tests.